ALAuto.py

```diff
@@ -219,7 +241,7 @@ def print_cycle_stats(self):
             script.print_cycle_stats()
         else:
             Logger.log_msg("Nothing to do, will check again in a few minutes.")
-            Utils.script_sleep(300)
+            Utils.script_sleep(60)
```

Comment:
Why?

Reply:
use this for local test, I did not realize that I sent it to the master branch, but I also see no reason not to decrease the default time, 300 is a very high value no?

Reply:
Ya, back when I set 5 minutes I didn't know that script_sleep randomized the time.